src/components/CtaLoadingDotsAnimation.tsx

```diff
@@ -36,7 +36,7 @@ const loadingAnimCss = css`
     }
 `;
 
-export const LoadingDots = (): React.ReactElement => {
+export const buildLoadingDots = (fillStyle = '#707070'): React.ReactElement => {
```

Comment:
Would it be more conventional maybe to have this continue to be named `LoadingDots` (it is a component after all, not a function which returns a component)? The `fillStyle` could be passed as an (optional) prop, which would allow it to be called in the normal way: `<LoadingDots fillStyle="#121212" />`. What do you think?

Reply:
Current code matches the factory approach we have to generating CSS, but I'm happy to turn this into a component. Will action.

Reply:
Done!